MANIFEST.in

```diff
@@ -1,31 +1,41 @@
-include MANIFEST.in
+# Reference: https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html
+
+# Include source files in SDist
 include CMakeLists.txt
+include *.bzl
+include *.bazel
+include WORKSPACE
+include requirements.txt
+include version.txt
+include .gitmodules
+graft android
+graft aten
+graft binaries
+graft c10
+graft caffe2
+graft cmake
+graft functorch
+graft third_party
+graft tools
+graft torch
+graft torchgen
+# FIXME: torch-xla codegen build failure if include this file in wheel
+exclude torchgen/BUILD.bazel
```

Comment:
`torch_xla/csrc/BUILD` genrule command (from target `//torch_xla/csrc:gen_lazy_tensor`) cannot import `torchgen` if this file is present in `site-packages/torchgen/BUILD.bazel`. cc @qihqi

The following fix will not resolve the import issue.

- pytorch/xla#9310

CI failure before pytorch/xla#9310: https://hud.pytorch.org/pr/pytorch/pytorch/153538#43672158222
CI failure after pytorch/xla#9310: https://hud.pytorch.org/pr/pytorch/pytorch/153538#44066727613